catkit2/services/hamamatsu_camera/hamamatsu_camera.py

```diff
@@ -270,8 +261,90 @@ def monitor_temperature(self):
             temperature = self.get_temperature()
             self.temperature.submit_data(np.array([temperature]))
 
+            if temperature > 30:
+                self.log.warning('Camera Temperature above 30 degrees, stopping acquisition and start fan')
+                self.fan_status = 'on'
+                self.should_shut_down = True
+
             self.sleep(0.1)
 
+    @property
+    def cooler_mode(self):
+        """
+        Check the cooling mode
+
+        Returns
+        -------
+        coolstr:
+            sensor cooler status 'on', 'off' or 'max'

```

Comment:
Would be nice to capitalize and add a period at the end like in the other docstrings.